packages/xo-web/src/xo-app/jobs/overview/index.js

```diff
@@ -32,6 +29,101 @@ const jobKeyToLabel = {
   genericTask: _('customJob'),
 }
 
+const COLUMNS = [
+  {
+    itemRenderer: schedule => {
+      const { id } = schedule
+
+      return (
+        <div>
+          <span>{`${schedule.name} (${id.slice(4, 8)})`}</span>
+          <Link
+            className='btn btn-sm btn-primary ml-1'
+            to={`/jobs/schedules/${id}/edit`}
+          >
+            <Icon icon='edit' />
+          </Link>
+        </div>
+      )
+    },
+    name: _('schedule'),
+    sortCriteria: 'name',
+  },
+  {
+    itemRenderer: (schedule, userData) => {
+      const job = userData.jobs[schedule.jobId]
+
+      return (
+        job !== undefined && (
+          <div>
+            <span>{`${job.name} - ${job.method} (${job.id.slice(4, 8)})`}</span>
+            <Link
+              className='btn btn-sm btn-primary ml-1'
+              to={`/jobs/${job.id}/edit`}
+            >
+              <Icon icon='edit' />
+            </Link>
+          </div>
+        )
+      )
+    },
+    name: _('job'),
+    sortCriteria: (schedule, userData) => {
+      const job = userData.jobs[schedule.jobId]
+      return job !== undefined && job.name
+    },
+  },
+  {
+    itemRenderer: schedule => schedule.cron,
+    name: _('jobScheduling'),
+  },
+  {
+    itemRenderer: schedule => (
+      <StateButton
+        disabledLabel={_('jobStateDisabled')}
+        disabledHandler={enableSchedule}
+        disabledTooltip={_('logIndicationToEnable')}
+        enabledLabel={_('jobStateEnabled')}
+        enabledHandler={disableSchedule}
+        enabledTooltip={_('logIndicationToDisable')}
+        handlerParam={schedule.id}
+        state={schedule.enabled}
+      />
+    ),
+    name: _('jobState'),
+  },
+  {
+    itemRenderer: (schedule, userData) =>
+      !userData.isScheduleUserMissing[schedule.id] && (
+        <Tooltip content={_('jobUserNotFound')}>
+          <Icon className='mr-1' icon='error' />
+        </Tooltip>
+      ),
+  },
+]
+
+const INDIVIDUAL_ACTIONS = [
+  {
+    disabled: (schedule, userData) =>
+      !userData.isScheduleUserMissing[schedule.id],
```

Comment:
Why are those maps names wrong? Am I missing something?
If not, please revert the `!!find` to `!find` in `_getIsScheduleUserMissing` and revert the conditions when the map is used.